roles/download/defaults/main.yml

```diff
@@ -60,6 +60,7 @@ pod_infra_version: 3.1
 contiv_version: 1.2.1
 cilium_version: "v1.3.0"
 kube_router_version: "v0.2.5"
+crictl_version: "{{ kube_version | regex_replace('^v(?P<major>\\d+).(?P<minor>\\d+).(?P<patch>\\d+)$', 'v\\g<major>.\\g<minor>.0') }}"
```

Comment:
I don't agree that assume last sub minor version is always 0.
Currently it's true, but there are no guarantee.
It should be 1.14.0 currently. Because master branch assume kube version is 1.14.1.
If user want to use another version, let user to set crictl_version manually.

Reply:
They can set it up manually, it's only the default.

Reply:
For the assumption, I could alternatively have dictionary mapping between kubernetes version and crictl version?

Reply:
I prefer manual configuration for crictl version and keep default crictl version for current k8s version. If user want to use old kubernetes, then user will change crictl_version manually.

Reply:
Done